package.json

```diff
@@ -29,24 +29,23 @@
     "lint": "lerna run lint --stream",
     "lint:changes": "yarn lint --since HEAD^",
     "lint:fix": "lerna run lint:fix --stream",
-    "lint:commit": "ui-scripts --lint-commit",
+    "lint:commit": "ui-scripts lint-commit",
     "lint:staged": "lint-staged",
     "update:package:list": "lerna run generate:package:list --stream --scope @instructure/instui-config",
     "commit:package:list": "git add packages/instui-config/package-lists/**/package-list.json",
     "generate:component": "yarn workspace @instructure/instui-cli instui create component",
     "generate:package": "yarn workspace @instructure/instui-cli instui create package",
-    "commit": "ui-scripts --commit",
+    "commit": "ui-scripts commit",
     "bootstrap": "time yarn node scripts/bootstrap.js",
     "build": "lerna run build --stream",
     "build:watch": "lerna run build:watch --stream",
     "build:docs": "lerna run bundle --stream --scope docs-app",
     "build:tokens": "ui-token-scripts generate-all-tokens",
     "build:types": "tsc -b tsconfig.references.json",
     "build:ts": "lerna run prepare-build --stream --scope @instructure/ui-icons && yarn build:types --verbose",
-    "build:node-scripts": "yarn workspace @instructure/ui-scripts build:ts",
```

Comment:
one of the main change is that ui-scripts is now plain JS so it does not need to be built